unstructured/ingest/cli/interfaces.py

```diff
@@ -184,3 +185,40 @@ def add_cli_options(cmd: click.Command) -> None:
             ),
         ]
         cmd.params.extend(options)
+
+
+class CliEmbeddingsConfig(EmbeddingConfig, CliMixin):
+    @staticmethod
+    def add_cli_options(cmd: click.Command) -> None:
+        options = [
+            click.Option(
+                ["--embedding-api-key"],
+                help="openai api key",
+                envvar="OPENAI_API_KEY",
```

Comment:
probably shouldn't hardcode openai naming assumption here, since in the future will be relevant to the chosen model / location